app/src/layouts/DashboardLayout/MenuHeader/WorkspaceSelector/index.jsx

```diff
@@ -71,13 +71,14 @@ const getWorkspaceIcon = (workspaceName) => {
 const WorkSpaceDropDown = ({ menu, hasNewInvites }) => {
   // Global State
   const user = useSelector(getUserAuthDetails);
-  const currentlyActiveWorkspace = useSelector(getCurrentlyActiveWorkspace);
-  const isWorkspaceMode = useSelector(getIsWorkspaceMode);
-  const isLocalWorkspace = useSelector(getIsWorkspaceLocal);
+  const activeWorkspaceId = useSelector(getActiveWorkspaceId);
+  const activeWorkspace = useSelector(getActiveWorkspace);
```

Comment:
could we skip creating `activeWorkspaceId` and instead use `activeWorkspace.id` ?

Reply:
Didn't want to look into small optimizations right now. So kept it same for now